CHANGELOG.rst

```diff
@@ -2,6 +2,191 @@
 Release Notes
 =============
 
+6.2.0
+=====
+2019-05-11
```

Comment:
2018 ? 🤔

Reply:
🤦‍♂️ my bad, let me fix this, thanks @koudaiii!

Reply:
Thank you 😄 I'm looking forward to 6.2.0 being released 👍 👍 👍